dev-tools/reconfigurator-cli/tests/output/cmd-example-stdout

```diff
@@ -91,15 +91,16 @@ parent:    02697f74-b14a-4418-90f0-c28b2a3a6aa9
     clickhouse        fe79023f-c5d5-4be5-ad2c-da4e9e9237e4   in service    fd00:1122:3344:102::23
     crucible          054f64a5-182c-4c28-8994-d2e082550201   in service    fd00:1122:3344:102::26
     crucible          3b5bffea-e5ed-44df-8468-fd4fa69757d8   in service    fd00:1122:3344:102::27
+    crucible          4f2eb088-7d28-4c4e-a27c-746400ec65ba   in service    fd00:1122:3344:102::2f
     crucible          53dd7fa4-899e-49ed-9fc2-48222db3e20d   in service    fd00:1122:3344:102::2a
     crucible          7db307d4-a6ed-4c47-bddf-6759161bf64a   in service    fd00:1122:3344:102::2c
     crucible          95ad9a1d-4063-4874-974c-2fc92830be27   in service    fd00:1122:3344:102::29
     crucible          bc095417-e2f0-4e95-b390-9cc3fc6e3c6d   in service    fd00:1122:3344:102::28
     crucible          d90401f1-fbc2-42cb-bf17-309ee0f922fe   in service    fd00:1122:3344:102::2b
     crucible          e8f994c0-0a1b-40e6-8db1-40a8ca89e503   in service    fd00:1122:3344:102::2d
     crucible          e9bf481e-323e-466e-842f-8107078c7137   in service    fd00:1122:3344:102::2e
-    crucible          f97aa057-6485-45d0-9cb4-4af5b0831d48   in service    fd00:1122:3344:102::25
-    crucible_pantry   eaec16c0-0d44-4847-b2d6-31a5151bae52   in service    fd00:1122:3344:102::24
+    crucible_pantry   f97aa057-6485-45d0-9cb4-4af5b0831d48   in service    fd00:1122:3344:102::25
+    external_dns      eaec16c0-0d44-4847-b2d6-31a5151bae52   in service    fd00:1122:3344:102::24
```

Comment:
It looks like these changes were just due to adding external DNS shifting the generated UUIDs around?

Reply:
Yes (and this is strongly related to your other comment, will follow up there)